standalone-metastore/metastore-common/pom.xml

```diff
@@ -564,29 +560,6 @@
         </executions>
       </plugin>
       <!-- TODO MS-SPLIT javadoc plugin -->
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-enforcer-plugin</artifactId>
-        <executions>
-          <execution>
-            <id>enforce-banned-dependencies</id>
-            <goals>
-              <goal>enforce</goal>
-            </goals>
-            <configuration>
```

Comment:
removed the enforcer pluging from `standalone-metastore/metastore-common/pom.xml` and `standalone-metastore/metastore-server/pom.xml` and moved to `standalone-metastore/pom.xml` i.e. a common place and updated it with the same list of banned dependency as in parent `pom.xml`